CeCILL-C.txt

```diff
@@ -0,0 +1,14 @@
+Copyright (C) 2006-2015 INRIA and contributors
```

Comment:
`LICENSE.txt` is a standard by some external tools. I don't think it is a good idea to change it. But, I can change `CeCILL-C.txt` by `LICENSE-short.txt` if you want.